wavefront_sdk/common/utils.py

```diff
@@ -261,3 +264,20 @@ def tracing_span_to_line_data(name, start_millis, duration_millis, source,
     str_builder.append(str(start_millis))
     str_builder.append(str(duration_millis))
     return ' '.join(str_builder) + '\n'
+
+
+def span_log_to_line_data(trace_id, span_id, span_logs, scrambler=None):
+    """Wavefront Tracing Span Log JSON format.
+
+    @param trace_id: Trace ID
+    @param span_id: Span ID
+    @param span_logs: Span Log
+    @param scrambler: Additional UUID, optional
+    @return: Span Log in JSON Format
+    """
+    span_log_json = {'traceId': str(trace_id),
+                     'spanId': str(span_id),
+                     'logs': span_logs}
+    if scrambler:
+        span_log_json['_scrambler'] = str(scrambler)
+    return json.dumps(span_log_json, default=lambda o: o.__dict__) + '\n'
```

Comment:
Just curious, what do we need `default` here for? Do we expect any non-string data to be processed?
The `.__dict__` for an arbitrary object would simply return all it's public attributes/methods

Reply:
We only support string data for span logs

Reply:
It means we don't need to override the default as any object from conversion table [0] is supported.
* [0] https://docs.python.org/3/library/json.html#py-to-json-table

Reply:
Won't work since SpanLog is a customized class, which is not JSON serializable. Will remain this way.